src/frontend/src/pages/error.jsx

```diff
@@ -72,7 +72,7 @@ const useStyles = makeStyles((theme) => ({
   fab: {
     position: 'relative',
     fontSize: '1.5rem',
-    color: 'white',
+    color: '#D3D3D3',
```

Comment:
Sorry, I should have been clearer. This "color: theme.palette.text.default" is what I meant.